janus_collector/examples/collect.rs

```diff
@@ -250,8 +251,7 @@ struct Options {
     #[clap(
         long,
         required = false,
-        takes_value = true,
-        multiple_occurrences = false,
+        num_args = 1,
         action = ArgAction::Set,
         value_parser = BucketsValueParser::new(),
         help_heading = "VDAF ALGORITHM AND PARAMETERS"
```

Comment:
We should title case the headings here as well, since the built-in ones changed to title case.